scalikejdbc-mapper-generator-core/src/main/scala/scalikejdbc/mapper/CodeGenerator.scala

```diff
@@ -376,6 +376,49 @@ class CodeGenerator(table: Table, specifiedClassName: Option[String] = None)(imp
         createColumns.map { c => 3.indent + c.nameInScala + " = " + c.nameInScala }.mkString(comma + eol) + ")" + eol +
         1.indent + "}" + eol
     }
+    /**
+     * {{{
+     * def batchInsert(list: List[Member])(implicit session: DBSession = autoSession) = {
+     *   val params: Seq[Seq[(Symbol, Any)]] = list.map(elem =>
+     *   Seq(
+     *     'id -> elem.id,
+     *     'name -> elem.name,
+     *     'birthday -> elem.birthday)
+     *   SQL("""insert into member (
+     *     id,
+     *     name,
+     *     birthday
+     *   ) values (
+     *     {id},
+     *     {name},
+     *     {birthday}
+     *   )""").batchByName(params: _*).apply()
+     * }
+     * }}}
+     */
+    val batchInsertMethod = {
+      val autoIncrement = table.autoIncrementColumns.size == 1
+      val batchInsertColumns: List[Column] =
+        if (autoIncrement)
+          allColumns.filterNot {
+            c => table.autoIncrementColumns.exists(_.name == c.name)
+          }
+        else
+          allColumns
+
+      // def batchInsert=(
+      1.indent + s"def batchInsert(list: List[" + className + "])(implicit session: DBSession = autoSession) {" + eol +
```

Comment:
```scala
(implicit session: DBSession = autoSession)
```

should be

```scala
(implicit session: DBSession$defaultAutoSession)
```


https://github.com/scalikejdbc/scalikejdbc/blob/2.2.7/scalikejdbc-mapper-generator-core/src/main/scala/scalikejdbc/mapper/CodeGenerator.scala#L200